Readme.md

```diff
@@ -221,7 +221,8 @@ $ sudo make install
 
     // Create handle
     rppHandle_t handle;
-    rppCreateWithBatchSize(&handle, noOfImages);
+    Rpp32u numThreads = 0;
```

Comment:
add explanation for numThreads parameter

Reply:
Done